google/cloud/aiplatform/pipeline_jobs.py

```diff
@@ -240,6 +240,12 @@ def run(
 
         _LOGGER.log_create_with_lro(self.__class__)
 
+        if not sync:
+            _LOGGER.info(
+                "Pipeline Job is scheduled to run and will be available to view once created:\n%s"
+                % self._dashboard_uri()
```

Comment:
This line will always throw because `_dashboard_uri` references `resource_name` which references `self._gca_resource.name` which doesn't exist at this point because the resource hasn't been created. Perhaps it could be synthesized from `self.job_id` but let's discuss offline on why `_dashboard_uri` below isn't printing.